src/app/api/contactInfo/route.js

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios'
+
+export async function POST(request) {
+  const body = await request.json()
+  const referer = request.headers.get('referer')
+
+  const formFields = {

```

Comment:
you can define this outside the component

Reply:
Isn't it already in the route.js?